packages/website/src/components/ErrorsViewer.tsx

```diff
@@ -46,30 +36,45 @@ function ErrorBlock({
   return (
     <div className={`admonition alert alert--${severityClass(item.severity)}`}>
       <div className="admonition-content">
-        <div className="row row--no-gutters">
-          <div className="col col--12">
+        <div className={clsx(!!item.fixer && styles.fixerContainer)}>
+          <div>
             {item.message} {item.location}
           </div>
-          {item.hasFixers && (
-            <div className="col col--12">
-              {item.fixers.map((fixer, index) => (
-                <div key={index} className={styles.fixer}>
-                  <span>&gt; {fixer.message}</span>
-                  <button
-                    className="button button--primary button--sm"
-                    disabled={isLocked}
-                    onClick={(): void => {
-                      fixer.fix();
-                      setIsLocked(true);
-                    }}
-                  >
-                    fix
-                  </button>
-                </div>
-              ))}
-            </div>
+          {item.fixer && (
+            <button
+              className="button button--primary button--sm"
+              disabled={isLocked}
+              onClick={(): void => {
+                item.fixer!.fix();
+                setIsLocked(true);
+              }}
+            >
+              fix
+            </button>
           )}
         </div>
+        {item.suggestions.length > 0 && (
+          <div>
+            {item.suggestions.map((fixer, index) => (
+              <div
+                key={index}
+                className={clsx(styles.fixerContainer, styles.fixer)}
+              >
+                <span>&gt; {fixer.message}</span>
+                <button
+                  className="button button--primary button--sm"
+                  disabled={isLocked}
+                  onClick={(): void => {
+                    fixer.fix();
+                    setIsLocked(true);
+                  }}
+                >
+                  fix
+                </button>
+              </div>
+            ))}
+          </div>
```

Comment:
Nit: maybe make a `<FixButton>` component for these, to DRY the two class name strings? Also that way you can pass it `item.fix` as a prop and not need the `!` above.